lib/toucan.js

```diff
@@ -159,24 +159,43 @@ async function deployTco2(contracts, vintageId) {
 }
 
 async function mintBatch(cBatches, batch, vintageId, admin, broker) {
+	let batchBroker = broker;
```

Comment:
you don't need to go this route. function parameters are not `const`, so you can directly assign to `broker`

Reply:
but what if some batches wanted to use the default broker, and some had an override?

Reply:
what I mean is you don't need to use the `batchBroker` variable as you can simply assign to `broker` if you want to override